experimental/dds/tree2/src/external-utilities/README.md

```diff
@@ -0,0 +1,6 @@
+# External Utilities
```

Comment:
since you added this, it should probably be included in in the top level tree 2 readme architecture diagram.